### PR TITLE
[14.0][FIX] website_no_crawler, config param id

### PR DESCRIPTION
**Pull request by GSLabIt**

### CODE REVIEW

**Comment by GSLabIt** [comment]
@pedrobaeza quick fix for param id, sorry my fault 😢

File: website_no_crawler/data/data.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8' ?>
 <odoo>
     <!-- Add add crawler configuration parameter -->
-    <record id="set_ribbon_color" model="ir.config_parameter">
+    <record id="set_crawler_disable" model="ir.config_parameter">
         <field name="key">webiste.crawler.disable</field>
```
Comment (pedrobaeza) [inline website_no_crawler/data/data.xml]: Shouldn't this be `website.crawler.disable` instead of `webiste.crawler.disable`?

Comment (pedrobaeza) [reply website_no_crawler/data/data.xml]: And why adding a system parameter?

Comment (GSLabIt) [reply website_no_crawler/data/data.xml]: sorry not getting the point 😬

Comment (GSLabIt) [reply website_no_crawler/data/data.xml]: got it, I'm blind ..

Comment (GSLabIt) [reply website_no_crawler/data/data.xml]: > And why adding a system parameter?

Added to allow setting based on environment using server env stuff

Comment (pedrobaeza) [reply website_no_crawler/data/data.xml]: I hate such options that overcomplicate all modules. Just don't install the module...

Comment (GSLabIt) [reply website_no_crawler/data/data.xml]: you're right, but if you have a restore from a db coming from another env, let's say from production to stage, you've to remember to uninstall it each time ..or vice versa

Comment (pedrobaeza) [reply website_no_crawler/data/data.xml]: Yes, but you automate that thing, not the other. On sysadmin side, bloating everything through environment variables makes difficult to follow a propagation problem or to trace things, and on Odoo side, to complicate code or to happen this kind of things.

Comment (GSLabIt) [reply website_no_crawler/data/data.xml]: well, as u prefer ..not going to argue ..do whatever u think is the best.

Comment (pedrobaeza) [reply website_no_crawler/data/data.xml]: Well, the module is already merged and approve by others, so I'm not going to undo it, but please take it into account for future "new features".

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

File: website_no_crawler/data/data.xml (hunk 1 above)
Comment (GSLabIt) [reply website_no_crawler/data/data.xml]: I'll surely do. Thanks

**Pull request merged**